src/Styleguide/Utils/BreakpointVisualizer.tsx

```diff
@@ -57,7 +57,9 @@ export class BreakpointVisualizer extends React.Component<
             <Responsive>
               {breakpoints => {
                 if (!isEqual(breakpoints, this.state.breakpoints)) {
-                  this.setState({ breakpoints })
+                  setImmediate(() => {
+                    this.setState({ breakpoints })
+                  })
```

Comment:
FYI @zephraph

Reply:
Btw, I was wondering if you could elaborate a bit on what this piece of the code is actually meant to do?

Reply:
The visualizer? It just makes it easier to see what breakpoint you're in...

![2018-07-25 15 04 36](https://user-images.githubusercontent.com/3087225/43221850-1c099180-901c-11e8-9298-95db49539fc3.gif)

It's intended as a development only component.

Reply:
Yeah that’s a very helpful widget! But I meant specifically this code that immediately re-renders with different breakpoints.